src/reanimated2/animation/springUtils.ts

```diff
@@ -52,6 +58,35 @@ export interface InnerSpringAnimation
   toValue: number;
   current: number;
 }
+export function isConfigValid(config: DefaultSpringConfig): boolean {
+  'worklet';
+  let errorMessage = '';
+  (
+    [
+      'stiffness',
+      'damping',
+      'dampingRatio',
+      'restDisplacementThreshold',
+      'restSpeedThreshold',
+      'mass',
+    ] as const
+  ).forEach((prop) => {
+    const value = config[prop];
+    if (value <= 0) {
+      errorMessage += `, ${prop} must be grater than zero but got ${value}`;
+    }
+  });
+
+  if (config.duration < 0) {
+    errorMessage += `, duration can't be negative, got ${config.duration}`;
+  }
+
+  if (errorMessage !== '') {
+    console.warn('[Reanimated] Invalid spring config' + errorMessage);
```

Comment:
The name of this method `isConfigValid` suggests that it has no side-effects, however it can call `console.warn`. Can we think of a better name here?

Reply:
I think that `checkIfConfigIsValid` may be better here.

Reply:
Sounds good